### DISCUSSION TITLE
GitHub Mobile settings headers not well distinguished

### DISCUSSION THREAD

**Discussion by caproven** [community/community#77358] [Mobile]
Using the latest GitHub Mobile app on Android, headers on the setting page are hardly distinguishable from content beneath them.

It is slightly better in dark mode since there's more font color contrast. My suggestion is to add a divider / line beneath the headers to convey separation of headers & content.

![Screenshot_20231129-141130.png](https://github.com/community/community/assets/18606244/6bf3520a-fe8d-4c81-a6b8-e9892ce1512b)

![Screenshot_20231129-141202.png](https://github.com/community/community/assets/18606244/5d9051ad-bcb5-4dcc-9a8d-455570aaebf5)

**Comment by HenSquared** [comment]
Thanks for the feedback! I have opened an issue with our Android team for them to discuss. :)
